go/gildedrose/Updater.go

```diff
@@ -0,0 +1,88 @@
+package gildedrose
+
+type Updater interface {
+	Update()
+}
+
+type TafkalItem struct {
+	Item
+}
+
+func NewTakfkalItem(item *Item) Updater {
```

Comment:
WTF is `Tafkal`?! And what is `Takfkal`? 

The item category is concert backstage tickets - re-read the requirements please